src/App/Pages/Accounts/TwoFactorPage.xaml

```diff
@@ -123,6 +122,12 @@
                     Margin="10, 20, 10, 10"
                     HorizontalTextAlignment="Center" />
             </StackLayout>
+            <Button Text="{u:I18n Continue}"
+                    IsEnabled="False"
+                    IsVisible="{Binding EmailMethod}"
```

Comment:
I think this should be bound to `ShowContinue`, right? Currently I'm missing the button when using the authenticator method.

Reply:
Oops, damn typo 😬